MobSF/utils.py

```diff
@@ -631,7 +631,7 @@ def is_safe_path(safe_root, check_path):
     safe_root = os.path.normpath(safe_root)
     check_path = os.path.normpath(check_path)
     return os.path.commonprefix(
-        (os.path.realpath(check_path), safe_root)) == safe_root
+        (os.path.realpath(check_path), os.path.realpath(safe_root))) == os.path.realpath(safe_root)
```

Comment:
I will suggest that you do this on the line before when variable is initialized.
Something like
```
safe_root = os.path.realpath(os.path.normpath(safe_root))
check_path = os.path.realpath(os.path.normpath(check_path))
```